Project.toml

```diff
@@ -11,22 +11,22 @@ PooledArrays = "2dfb63ee-cc39-5dd5-95bd-886bf059d720"
 SentinelArrays = "91c51154-3ec4-41a3-a24f-3f23e20d615c"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
+WeakRefStrings = "ea10d353-3f73-51f8-a26c-33c1cb351aa5"
 
 [compat]
 FilePathsBase = "0.6,0.7,0.8,0.9"
 Parsers = "1"
 PooledArrays = "0.5, 1.0"
 SentinelArrays = "1.2"
 Tables = "1.1"
-WeakRefStrings = "0.5,0.6"
-julia = "1"
+WeakRefStrings = "0.5,0.6,1"
+julia = "1.3"
```

Comment:
so CSV.jl will not work on Julia 1.0 after these changes?

Reply:
I'll have to review; I think there was a dependency or two that needed 1.3

Reply:
Yes, with the SentinelArrays.jl ChainedVector code, there were a lot of updates/changes in array functions pre 1.3 that were a hassle to try and support, so SentinelArrays.jl is now 1.3